### PR TITLE
Clear endpoints for cell type marker gene heatmap

### PR DESCRIPTION
**Pull request by lingyun1010**
Remove the getting cell types endpoint and refactor cell type marker gene heatmap data endpoint by removing cell type variables.

### CODE REVIEW

File: app/src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGeneService.java [hunk 1]
```diff
@@ -52,13 +52,12 @@ public ImmutableList<String> getCellTypesWithMarkerGenes(String experimentAccess
     }
 
     public ImmutableList<MarkerGene> getCellTypeMarkerGeneHeatmapData(String experimentAccession,
-                                                                      String cellGroupType,
-                                                                      ImmutableCollection<String> cellTypes) {
+                                                                      String cellGroupType) {
         var cellTypeMarkerGenes =
                 markerGenesDao
-                        .getCellTypeMarkerGenesOntologyLabels(
+                        .getCellTypeMarkerGenes(
```
Comment (upendrakumbham) [inline app/src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGeneService.java:58]: can you refactor tests as well?

**Review by alfonsomunozpomer — CHANGES_REQUESTED** [review]
Please rebase on top of `develop` and get this branch tested.

**Review by alfonsomunozpomer — APPROVED** [review]
All good. You’ll need to merge again `develop`. Sorry about that! It shouldn’t be much of a problem, though.

**Review by upendrakumbham — APPROVED** [review]
ALGTM!

**Pull request merged**